lldb/source/API/SBDebugger.cpp

```diff
@@ -1686,13 +1686,33 @@ void SBDebugger::SetLoggingCallback(lldb::LogOutputCallback log_callback,
   }
 }
 
-void SBDebugger::SetDestroyCallback(
-    lldb::SBDebuggerDestroyCallback destroy_callback, void *baton) {
+lldb::SBDebuggerDestroyCallbackToken
+SBDebugger::AddDestroyCallback(lldb::SBDebuggerDestroyCallback destroy_callback, void *baton) {
+  LLDB_INSTRUMENT_VA(this, destroy_callback, baton);
+  if (m_opaque_sp) {
+    return m_opaque_sp->AddDestroyCallback(
+        destroy_callback, baton);
+  }
+  return -1;
```

Comment:
It's better to make an LLDB_INVALID_DESTROY_TOKEN and use that than have a magical -1 floating around in code.

Reply:
Added `LLDB_INVALID_DESTROY_CALLBACK_TOKEN`.